servers/audio_server.h

```diff
@@ -62,6 +62,7 @@ class AudioDriver {
 	unsigned int input_size = 0;
 
 	void audio_server_process(int p_frames, int32_t *p_buffer, bool p_update_mix_time = true);
+	void audio_server_init_channels_and_buffers();
```

Comment:
Exposed it, since Oboe doesn't recommend to allocate memory in callbacks, which would happen through audio_server_process if device changed